common/src/main/java/com/wynntils/models/wynnitem/type/ItemObtainType.java

```diff
@@ -4,16 +4,21 @@
  */
 package com.wynntils.models.wynnitem.type;
 
+import java.util.List;
+
 public enum ItemObtainType {
     // From Wynncraft API
     LOOT_CHEST("lootchest", "Tier III/IV Loot Chest"), // lootchests (implies t3 or t4, afaict)
     NORMAL_MOB_DROP("normal", "Unspecified Mob Drop"), // mob drops (and any loot chest, afaict)
+    CHALLENGE("challenge", "Challenge"), // at the moment, only Legendary Island
+    EVENT("event", "Event"), // like Bonfire, Heroes, etc.
+    LOOTRUN("lootrun", "Lootrun"), // lootrun
     UNKNOWN("never", "Unknown"), // more information is needed
 
-    // From crowd sourced data
-    BOSS_ALTAR("bossaltar", "Boss Altar"),
+    // From crowd sourced data or Wynncraft API
+    BOSS_ALTAR(List.of("bossaltar", "altar"), "Boss Altar"),
```

Comment:
It's a bit sad if our data has a different key. At some point, we should update it to match Wynn's tag, so we don't need this list. Also, perhaps Wynn's list of obtain info is more complete now? Do we really still need our crowd sourced data?

Reply:
I tried to get rid of the crowd-sourced, partially outdated data, but it is still much better than the info API provides, so we can not get away with it.